systemvm/patches/debian/config/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -261,20 +261,9 @@ def set_master(self):
 
         self.set_lock()
         logging.debug("Setting router to master")
-        ads = [o for o in self.address.get_ips() if o.is_public()]
-        dev = ''
-        for o in ads:
-            if dev == o.get_device():
-                continue
-            cmd2 = "ip link set %s up" % o.get_device()
-            if CsDevice(o.get_device(), self.config).waitfordevice():
-                CsHelper.execute(cmd2)
-                dev = o.get_device()
-                logging.info("Bringing public interface %s up" %
-                             o.get_device())
-            else:
-                logging.error(
-                    "Device %s was not ready could not bring it up" % o.get_device())
+        self.address.process()
+        logging.info("added default rotue")
```

Comment:
There is a typo here. Could you fix that?

Also, the logging could be more verbose. To what is the default route pointing? We should log that as well.